### PR TITLE
Update adoptawidget page with list of contributions

### PR DESCRIPTION
**Pull request by johnpryan**
Staged: https://flutter-website-jr-stagi-988d4.firebaseapp.com/adoptawidget

### CODE REVIEW

File: src/adoptawidget/index.md [hunk 1]
```diff
@@ -12,6 +12,29 @@ helps everyone! So, for the month of November, we’re extending a special
...
+- [CaptainIRS](https://github.com/CaptainIRS) contributed to the [AbsorbPointer docs](https://master-api.flutter.dev/flutter/widgets/AbsorbPointer-class.html) ([pull request](https://github.com/flutter/flutter/pull/69521))
+- [abd99](https://github.com/abd99) contributed to the [Dismissible docs](https://master-api.flutter.dev/flutter/widgets/Dismissible-class.html) ([pull request](https://github.com/flutter/flutter/pull/69535))
+- [redsolver](https://github.com/redsolver) contributed to the [Table docs](https://master-api.flutter.dev/flutter/widgets/Table-class.html) ([pull request](https://github.com/flutter/flutter/pull/69588))
+- [tsvillain](https://github.com/tsvillain) contributed to the [Tooltip](https://master-api.flutter.dev/flutter/material/Tooltip-class.html) ([pull request](https://github.com/flutter/flutter/pull/69518))
+- [Thesmader](https://github.com/Thesmader) contributed to the [SnackBar](https://master-api.flutter.dev/flutter/material/SnackBar-class.html) ([pull request](https://github.com/flutter/flutter/pull/69555))
+
```
Comment (sfshaza2) [inline src/adoptawidget/index.md:37]: @johnpryan, this looks pretty good, but I feel like it's missing some text. How about adding something like:

Thanks to your eager participation, all of the widgets were assigned in xxx hours and all issues have been addressed, though we are still reviewing PRs. Based on the success of this experiment, we may do something similar in the future, so stay tuned!

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
thx!

**Comment by johnpryan** [comment]
@sfshaza2 this is ready for another look

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**